src/scripts/pokemons/PokemonList.ts

```diff
@@ -14430,7 +15019,11 @@ const pokemonList = createPokemonArray(
         'levelType': LevelType.mediumfast,
         'exp': 56,
         'catchRate': 190,
-        'evolutions': [new LevelEvolution('Rockruff', 'Lycanroc', 25)],
+        //'evolutions': [
+        //    new DayTimedLevelEvolution('Rockruff', 'Lycanrock (Midday)', 25),
+        //    new NightTimedLevelEvolution('Rockruff', 'Lycanrock (Midnight)', 25),
+        //    new DuskTimedLevelEvolution('Rockruff', 'Lycanrock (Dusk)', 25),
+        //],
```

Comment:
Are these supposed to be commented out?

Reply:
They are commented for now because DuskTimedLevelEvolution does not exist yet.